networkx/drawing/tests/test_pylab.py

```diff
@@ -250,3 +250,10 @@ def test_alpha_iter(self):
     def test_error_invalid_kwds(self):
         with pytest.raises(ValueError, match="Received invalid argument"):
             nx.draw(self.G, foo="bar")
+
+    @pytest.mark.skipif(os.environ.get("OPTIONAL_DEPS") != "1",
```

Comment:
This ```skipif``` fature is something I didn't know about. Thanks for that!
But there is a tool more specifically for this case that I'd rather use.
(I'd rather not have to set my environment variables when testing locally.)

It's called [pytest.importorskip](https://docs.pytest.org/en/stable/reference.html#pytest-importorskip). Basically you try to import and ifthe import doesn't work it skips this test.  So, here you would remove the decorator and replace the import line with:
```python
        np = pytest.importorskip("numpy")
```

Reply:
Done